src/index.html

```diff
@@ -8,9 +8,45 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="form big-form" 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post" 
+      data-qa="big"
+    > 
+      <input 
+        class="search big-search" 
+        type="text" 
+        name="bigSearch" 
+        data-qa="keypress" 
+        placeholder="Try &quot;Los Angeles&quot;"
+      > 
+      <img 
+        src="Search.svg" 
+        class="icon1" 
```

Comment:
```suggestion
        class="icon-big" 
```
FYI (for the future)